File: negative_move/neg.js
var stdin= process.openStdin();
stdin.addListener("data", function(d)
{
var x= d.toString().trim().split("\n");
let b= Number(x[0]);
let sub = JSON.parse(x[1]);
var res= rearrange(sub,b);
let s='';
for(var i=0;i<b;i++)
{
s+=res[i]+" ";
}
console.log(s);
});



function rearrange(arr,n)
{
	var alternate_array=[0];
    //Write your code here without removing the exising code
	//the variable 'arr' contains array of integers of size n.
	//modified the array 'alternate_array' having alternate positive and negative numbers.
	return alternate_array;
    
}

File: negative_move/neg.java

	
	public static void main(String[] args)
    {
			Scanner sc=new Scanner(System.in);
            int n = sc.nextInt(); 
			int []arr= Arrays.stream(sc.next().split("[,\\[\\]]")).filter(w-> !w.equals("")).mapToInt(Integer::parseInt).toArray();
		    Solution g=new Solution();
            int []ans=g.sort_array(arr,n);  
            for(int i=0;i<n;i++)
            System.out.print(ans[i] + " ");  
          
	}
}
class Solution
{
	public static int[] sort_array(int arr[],int n)
        {
            int[] sorted_array = new int[n];
			//Write your code here without removing the exising code
	        //the variable 'arr' contains array of integers of size n.
	        //modified the array 'alternate_array' having alternate positive and negative numbers.
			return sorted_array;
        }
}